apps/mobile/src/app/confirm-transaction.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import { ConfirmTxContainer } from '@/src/features/ConfirmTx'
+
+function ConfirmTransactionPage() {
+  return (
+    <SafeAreaView style={{ flex: 1 }} edges={['bottom']}>
```

Comment:
Btw, I don't know if you've seen this doc: https://reactnavigation.org/docs/handling-safe-area/#use-the-hook-for-more-control, but the recommendation is to use the hook instead of the `<SafeAreaView`.

Reply:
hmm I dind't! Will take a look, thanks